pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/AggregationFunctionFactory.java

```diff
@@ -45,10 +45,12 @@ private AggregationFunctionFactory() {
   public static AggregationFunction getAggregationFunction(FunctionContext function, QueryContext queryContext) {
     try {
       String upperCaseFunctionName = function.getFunctionName().toUpperCase();
+      AggregationFunctionType aggregationFunctionType =
+          AggregationFunctionType.getAggregationFunctionType(upperCaseFunctionName);
       List<ExpressionContext> arguments = function.getArguments();
       ExpressionContext firstArgument = arguments.get(0);
-      if (upperCaseFunctionName.startsWith("PERCENTILE")) {
-        String remainingFunctionName = upperCaseFunctionName.substring(10);
+      if (aggregationFunctionType.getName().toUpperCase().startsWith("PERCENTILE")) {
```

Comment:
Why do we have to do upperCase again?
AggregationFunctionType is an enum so getName() should return upper case right?

Reply:
We defined enums as:
```
  PERCENTILEMV("percentileMV"),
  PERCENTILEESTMV("percentileEstMV"),
  PERCENTILETDIGESTMV("percentileTDigestMV"),
```

Reply:
`.name()` will return upper case though :-P